test/spanner/config/validator_test.exs

```diff
@@ -17,10 +17,68 @@ defmodule Spanner.Config.Validator.Test do
     }
   end
 
+  defp enforcing_config do
+    %{"name" => "foo",
+      "version" => "0.0.1",
+      "commands" => %{
+        "bar" => %{
+          "executable" => "/bin/bar",
+          "enforcing" => true}}}
+  end
```

Comment:
I thought about doing what you suggest but decided implicitly coupling the test datasets would do more harm than good in the long term.